docs/source/using-executorch-export.md

```diff
@@ -125,6 +125,31 @@ with open("model.pte", "wb") as file:
 
 This yields a `model.pte` file which can be run on mobile devices.
 
+To generate a `model.pte`, `model.ptd` pair with the weights inside `model.ptd`, add the following transform function to tag constants as external:
+
+```python
+from executorch.exir.passes.external_constants_pass import (
+    delegate_external_constants_pass,
+)
+partial_function = partial(
+    delegate_external_constants_pass,
+    ep=exported_program,
+    gen_tag_fn=lambda x: "model", # This is the filename the weights will be saved to.
```

Comment:
"This is the filename the weights will be saved to. In this case, weights will be saved as "model.ptd""